app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :user_signup_params, :resource_name
```

Comment:
Is it mandatory to call `:resource_name`?

Reply:
actually, not. It works without it. Removed.